modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java

```diff
@@ -112,7 +112,7 @@ public Object parse(ElementInstance instance, Node node, Object value)
     
     @Override
     public Object getProperty(Object object, QName name) throws Exception {
-        if (GML.geometryMember.equals(name)) {
+        if ("geometryMember".equals(name.getLocalPart())) {
```

Comment:
Yes, I followed the same pattern of all the other classes which use literals.
But I agree that would be better.